src/pageObject.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = this.header;
```

Comment:
You are assigning the value of 'this.header' to 'this.header' which is unnecessary and redundant.

Reply:
fixed